send.go

```diff
@@ -75,9 +75,20 @@ func (client *SQS) sendMessage(input *sqs.SendMessageInput) (*sqs.SendMessageOut
 	return &sqs.SendMessageOutput{}, errorNonExistentQueue()
 }
 
-// SendMessageWithContext is not implemented. It will panic in all cases.
-func (client *SQS) SendMessageWithContext(aws.Context, *sqs.SendMessageInput, ...request.Option) (*sqs.SendMessageOutput, error) {
-	panic("SendMessageWithContext is not implemented")
+// SendMessageWithContext is partially supported. The following are not supported:
+//
+// - Recording ctx
+//
+// - Recording opts
+//
+// - Also see all features not supported for SendMessage()
+func (client *SQS) SendMessageWithContext(ctx aws.Context, input *sqs.SendMessageInput, opts ...request.Option) (*sqs.SendMessageOutput, error) {
+	client.httpRequest()
```

Comment:
I would prefer this simply proxies to the same implementation:

```go
func (client *SQS) SendMessageWithContext(ctx aws.Context, input *sqs.SendMessageInput, opts ...request.Option) (*sqs.SendMessageOutput, error) {
    return client.SendMessage(input)
}
```

Or, technically `SendMessage` should call `SendMessageWithContext` since `SendMessage` is technically a subset of `SendMessageWithContext`.

Reply:
Sorry, it has taken me so long to respond to this. I was away and didn't see your message until just now. 

Fixed by b1460a8a1a6993d097564ccd1901c2ceb91ba0fd